packages/contracts/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Do we need license file here? The one in root is sufficient right?
Is this to have LICENSE be par of npm packages?

Reply:
Yes exactly.
They're here to be delivered in the npm build, it's best practice.